cypress/e2e/tests/pages/manager/jwt-authentication.spec.ts

```diff
@@ -0,0 +1,169 @@
+
+import JWTAuthenticationPagePo from '@/cypress/e2e/po/pages/cluster-manager/jwt-authentication.po';
+import HomePagePo from '@/cypress/e2e/po/pages/home.po';
+
+describe('JWT Authentication', { testIsolation: 'off', tags: ['@manager', '@adminUser', '@jenkins'] }, () => {
+  const jwtAuthenticationPage = new JWTAuthenticationPagePo();
+
+  let instance0 = '';
+  let instance1 = '';
+  let removeCluster0 = false;
+  let removeCluster1 = false;
+  const region = 'us-west-1';
+  const namespace = 'fleet-default';
+
+  before(() => {
+    cy.login();
+    cy.createE2EResourceName('rke2cluster0').as('rke2Ec2ClusterName0');
+    cy.createE2EResourceName('rke2cluster1').as('rke2Ec2ClusterName1');
+
+    cy.get<string>('@rke2Ec2ClusterName0').then((name) => {
+      instance0 = name;
+      // create real cluster
```

Comment:
Do these tests really need to use actual third-party compute resources? It looks like the feature works with RKE2 custom clusters that don't have any nodes registered. Not a huge deal, just thinking about cost optimization

Reply:
Good catch, thank you